src/common/LoginModal.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+
+import './css/modal.css'
+class LoginModal extends Component {
+
+  loginModal() {
+    const modal = document.getElementById('modal');
+    modal.style.display = 'none';
+  }
+  render () {
+    return (
+      <div id="modal" onClick={this.loginModal}>
```

Comment:
Does this work or does it keep firing off?
I know in react native you usually use an arrow function for the onPress like this:
onPress={() => this.function}

Just wondering